Authentication/SignUp/TermsAndServices/TermsAndServicesController.swift

```diff
@@ -48,18 +47,18 @@ public final class TermsAndServicesController: UIViewController, WKNavigationDel
     
     public override func viewDidLoad() {
         super.viewDidLoad()
-        let request = NSURLRequest(URL: _url)
-        _webView.loadRequest(request)
+        let request = URLRequest(url: _url)
+        _webView.load(request)
         _delegate.didStartLoadingTermsAndServices(self)
     }
     
-    public override func viewWillAppear(animated: Bool) {
+    public override func viewWillAppear(_ animated: Bool) {
         super.viewWillAppear(animated)
-        navigationController?.navigationBarHidden = false
+        navigationController?.isNavigationBarHidden = false
     }
         
     // Called when a mainframe load completes.
```

Comment:
Nothing to do with migration, but if not let's add a card for the following:
Extract this function to an extension:
```
extension TermsAndServicesController: WKNavigationDelegate {
     //The function
}
```
and remove the protocol conformation from the upper declaration of the controller.